domenetjenester/inngangsvilkar/src/main/java/no/nav/ung/sak/inngangsvilkår/alder/VurderAldersVilkårTjeneste.java

```diff
@@ -1,49 +1,38 @@
 package no.nav.ung.sak.inngangsvilkår.alder;
 
-import java.time.LocalDate;
-import java.time.temporal.TemporalAdjusters;
-import java.util.NavigableSet;
-
-import no.nav.fpsak.tidsserie.LocalDateInterval;
-import no.nav.fpsak.tidsserie.LocalDateSegment;
-import no.nav.fpsak.tidsserie.LocalDateTimeline;
 import no.nav.ung.kodeverk.vilkår.Avslagsårsak;
 import no.nav.ung.kodeverk.vilkår.Utfall;
 import no.nav.ung.sak.behandlingslager.behandling.vilkår.VilkårBuilder;
 import no.nav.ung.sak.behandlingslager.behandling.vilkår.periode.VilkårPeriodeBuilder;
 import no.nav.ung.sak.domene.typer.tid.DatoIntervallEntitet;
-import no.nav.ung.sak.domene.typer.tid.TidslinjeUtil;
+
+import java.time.LocalDate;
+import java.time.temporal.TemporalAdjusters;
+import java.util.NavigableSet;
 
 public class VurderAldersVilkårTjeneste {
 
     public void vurderPerioder(VilkårBuilder vilkårBuilder, NavigableSet<DatoIntervallEntitet> perioderTilVurdering, LocalDate fødselsdato) {
-        var førsteDagMedGodkjentAlder = fødselsdato.plusYears(18).with(TemporalAdjusters.lastDayOfMonth()).plusDays(1);
-        var sisteDagMedGodkjentAlder = fødselsdato.plusYears(29).with(TemporalAdjusters.lastDayOfMonth());
+        var førsteDagMedGodkjentAlder = fødselsdato.plusYears(18);
+        var sisteDagMedGodkjentAlder = fødselsdato.plusYears(29).minusDays(1);
```

Comment:
Er det ikke til og med 29 år?

Reply:
Never mind :) 
Ser ut som det er til dagen før de fyller 29 år. Så det er riktig.

Reply:
Nei

Reply:
§ 3 Vilkår for tildeling av ungdomsprogrammet
Ungdomsprogrammet kan tildeles personer som
a) har fylt 18 år og ikke har fylt 29 år,
b) har fått fastslått at de har behov for bistand fra Arbeids- og velferdsetaten for å
komme i ordinært arbeid,
c) ikke tidligere har deltatt i programmet,
d) ønsker å delta i programmet,
e) vurderes å være i stand til å delta i programmet på fulltid og
f) som gjennom deltakelse i programmet vurderes å kunne bli i stand til å komme i
ordinært arbeid eller ordinær utdanning.
Deltakelse i programmet skal være nødvendig og hensiktsmessig for at deltakeren skal
komme i ordinært arbeid eller ordinær utdanning. Arbeids- og velferdsetaten avgjør hvem
som skal tildeles ungdomsprogrammet.